packages/cli-kit/src/private/node/ui/components/GitDiff.tsx

```diff
@@ -0,0 +1,47 @@
+import {unstyled, shouldDisplayColors} from '../../../../public/node/output.js'
+import {Text} from 'ink'
+import React, {FunctionComponent} from 'react'
+import {createRequire} from 'module'
+
+const require = createRequire(import.meta.url)
+const gitDiff = require('git-diff')
+
+export interface GitDiffProps {
+  baselineContent: string
+  updatedContent: string
+}
+
+/**
+ * `FilePath` displays a path to a file.
```

Comment:
This might be a remnant of a copy paste.

Reply:
Fixed the copypaste. Is the example necessary? I can't find any other place where we do this, so I'm not even sure about the right format.

Reply:
Not really necessary, just a nice addition. You could do something like

```
@example
@@ -1,2 +1,2 @@
    - hello
       world
    + hello
```